src/plugins/console/public/application/containers/editor/monaco/monaco_editor_actions_provider.ts

```diff
@@ -184,7 +184,7 @@ export class MonacoEditorActionsProvider {
   public async sendRequests(dispatch: Dispatch<Actions>, context: ContextValue): Promise<void> {
     const {
       services: { notifications, trackUiMetric, http, settings, history, autocompleteInfo },
-      startServices,
```

Comment:
Some of the changes in this PR shuffle around the structure of services provided by `useServicesContext`, and makes the top-level directly include `analytics`, `i18n`, and `theme` rather than namespacing them under `startServices`. This will be consistent with most of the other changes I've made as part of https://github.com/elastic/kibana/pull/180819